site/content/en/latest/user/quickstart.md

```diff
@@ -20,7 +20,7 @@ so the `Gateway` resource has an Address associated with it. We recommend using
 Install the Gateway API CRDs and Envoy Gateway:
 
 ```shell
-helm install eg oci://docker.io/envoyproxy/gateway-helm --version v0.0.0-latest -n envoy-gateway-system --create-namespace
```

Comment:
this quickstart is under `latest/user` directory, we should keep the version unchanged

Reply:
sorry, I was careless. corrected it.